test/cypress/integration/simplified-account/service-settings/webhooks/webhooks.cy.js

```diff
@@ -0,0 +1,124 @@
+
+const EXISTING_WEBHOOKS = [
+  {
+    external_id: WEBHOOK_EXTERNAL_ID,
+    callback_url: CALLBACK_URL,
+    description: DESCRIPTION,
+    subscriptions: SUBSCRIPTIONS,
+    status: 'ACTIVE'
+  },
+  {
+    status: 'ACTIVE'
+  },
+  {
+    status: 'INACTIVE'
+  }
+]
+
+const setStubs = (opts = {}, additionalStubs = []) => {
+  cy.task('setupStubs', [
+    userStubs.getUserSuccess({
+      userExternalId: USER_EXTERNAL_ID,
+      gatewayAccountId: GATEWAY_ACCOUNT_ID,
+      serviceName: SERVICE_NAME,
+      serviceExternalId: SERVICE_EXTERNAL_ID,
+      role: ROLES[opts.role || 'admin'],
+      features: 'degatewayaccountification' // TODO remove features once simplified accounts are live
+    }),
+    gatewayAccountStubs.getAccountByServiceIdAndAccountType(SERVICE_EXTERNAL_ID, LIVE_ACCOUNT_TYPE, {
+      gateway_account_id: GATEWAY_ACCOUNT_ID,
+      type: LIVE_ACCOUNT_TYPE,
+      provider_switch_enabled: opts.providerSwitchEnabled || false
+    }),
+    ...additionalStubs])
+}
+
+describe('for an admin', () => {
+  beforeEach(() => {
+    cy.setEncryptedCookies(USER_EXTERNAL_ID)
+    setStubs({}, [
+      getWebhooksListSuccess({ service_id: SERVICE_EXTERNAL_ID, gateway_account_id: GATEWAY_ACCOUNT_ID, live: true, webhooks: EXISTING_WEBHOOKS })
+    ])
+    cy.visit(WEBHOOKS_SETTINGS_URL)
+  })
+
+  it('should show webhooks in the setting navigation', () => {
+    cy.get('.service-settings-nav')
+      .find('li')
+      .contains('Webhooks')
+      .then(li => {
+        cy.wrap(li)
+          .should('have.attr', 'href', WEBHOOKS_SETTINGS_URL)
+          .parent().should('have.class', 'service-settings-nav__li--active')
+      })
+  })
+
+  it('should show title, heading and create Webhook button', () => {
+    cy.title().should('eq', 'Webhooks - Settings - McDuck Enterprises - GOV.UK Pay')
+    cy.get('h1').should('have.text', 'Webhooks')
+    cy.get('div.service-settings-pane')
+      .find('a')
+      .contains('Create a new webhook')
+      .should('have.attr', 'href', `${WEBHOOKS_SETTINGS_URL}/create`)
+  })
+
+  it('should show webhooks as summary cards', () => {
+    cy.get('div.govuk-summary-card').should('have.length', 3)
+    cy.get('div.govuk-summary-card').first().find('h2').first().contains(DESCRIPTION).get('strong.govuk-tag--blue').contains('Active')
+    cy.get('div.govuk-summary-card').first().find('a').first().contains('View')
+    cy.get('div.govuk-summary-card').first().find('a').eq(1).contains('Update')
+    cy.get('div.govuk-summary-card').first().find('dd').first().contains(CALLBACK_URL)
+    cy.get('div.govuk-summary-card').first().find('dd').eq(1).contains('Payment succeeded')
+    cy.get('div.govuk-summary-card').first().find('dd').eq(1).contains('Payment captured')
+    cy.get('div.govuk-summary-card').eq(1).find('h2').first().get('strong.govuk-tag--blue').contains('Active')
+    cy.get('div.govuk-summary-card').eq(2).find('h2').first().get('strong.govuk-tag--yellow').contains('Inactive')
```

Comment:
not super important since it'll still fail if these don't exist but these aren't strictly assertions, it's just trying to find these things in the DOM, adding `should('exist')` on the end of each makes them proper assertions - or equally the `.contains(...)` can be `.should('contain.text', ...)`

Reply:
may as well add a few more assertions for the other cards, particularly the inactive webhook card, just to check it shows everything it should

Reply:
thanks - I've added properties for the other two webhooks and asserted on them